src/components/Menu/index.tsx

```diff
@@ -67,13 +67,12 @@ export default class Menu extends React.Component<MenuProps, MenuState> {
 
   render() {
     let showVersionSwitcher = false;
-    if (
-      (this.props.url.startsWith("/ui") ||
-        this.props.url.startsWith("/ui-legacy")) &&
-      this.props.filterKey !== "react-native" &&
-      this.props.filterKey !== "flutter"
+    let showLibVersionSwitcher = false
+    if ((this.props.url.startsWith("/ui") || this.props.url.startsWith("/ui-legacy")) && this.props.filterKey !== "react-native" && this.props.filterKey !== "flutter"
     ) {
       showVersionSwitcher = true;
+    } else if ((this.props.url.startsWith("/lib") || this.props.url.startsWith("/lib-legacy")) && this.props.filterKey == 'ios') {
```

Comment:
```suggestion
    } else if ((this.props.url.startsWith("/lib") || this.props.url.startsWith("/lib-v1")) && this.props.filterKey == 'ios') {
```

Reply:
Thanks for catching that, updated to `lib-v1` in latest revision